tests/core/engine_adapter/test_spark.py

```diff
@@ -445,7 +445,147 @@ def test_scd_type_2(
         "CREATE TABLE IF NOT EXISTS `db`.`temp_target_abcdefgh` AS SELECT `id`, `name`, `price`, `test_updated_at`, `test_valid_from`, `test_valid_to` FROM `db`.`target`",
         "TRUNCATE TABLE `db`.`target`",
         parse_one(
-            """INSERT INTO `db`.`target` (
+            """WITH `source` AS (
```

Comment:
I think this is just a SQLGlot change. When I first generated this SQLGlot put the with statement in CTE but that wasn't compatible with Spark 2.X so SQLGlot changed to this format. Either way I think it would get transpiled so that is why it worked before.